README.md

```diff
@@ -133,61 +220,17 @@ you can open your IDE preferences (`PHPStorm -> Preferences`) and search for `Fi
 PHP Class Doc Comment, PHP File Header, PHP Interface Doc Comment 
 are those templates that should at least be customized.
 
-16. Use Behat Contexts are divided into `Hooks` - generic app Background, `Setup` specific resource background, `Ui` - specific interaction.
-17. Commit messages should be written (if only it's possible) with the following convention:
-`[Project spec state][Bundle] max 64 characters description in english written in Present Simple.`
-18. If there is an opened issue on Jira for a specific task, your branch should be named `sit_[ISSUE_NUMBER]`. If not, it should be named with the first letter of your name and your surname. In my case (Mikołaj Król) it would be `mkrol`.
-19. Be careful with using static fields.
-20. Be more careful when you think Singleton is something you need in project.
-21. Laravel is not that bad.
```

Comment:
Hey! Don't remove this one!

Reply:
I can't find a proper section for this one :D